services/spar/test-integration/Main.hs

```diff
@@ -40,8 +40,8 @@ import System.Random (randomRIO)
 import Test.Hspec
 import Test.Hspec.Core.Format
 import Test.Hspec.Core.Runner
-import Test.Hspec.JUnit
 import Test.Hspec.JUnit.Config.Env
+import Test.Hspec.JUnit.Formatter as JUnit
```

Comment:
This is awesome - they deprecate the module but don't offer alternatives for the functions provided from there ... tempted to just ignore the deprecation warnings. what do you think @akshaymankar

Reply:
I think you're supposed to use `Test.Hspec.JUnit.Format.junit`

Reply:
ahah, that makes sense, thank you <3